test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -609,6 +617,12 @@ test
     const deviceChallengePollPageObject = new DeviceChallengePollPageObject(t);
     await t.expect(deviceChallengePollPageObject.getHeader()).eql('Sign in with Okta FastPass');
 
+    await t.expect(deviceChallengePollPageObject.getSpinner().getStyleProperty('display')).eql('block');
```

Comment:
add a test the check for `hide` ?

Reply:
I think checking the `display` property is more reliable, `hide` is just the class to change the display property anyway.

Reply:
I meant to add a test for a screen that doesn't want to show it..